smk-edit/static/components/edit-tool-details-geomark.html

```diff
@@ -0,0 +1,19 @@
+<div>
+    <edit-tool-details
+        v-bind:toolType="toolType"
+        v-bind:toolInstance="toolInstance"
+    ></edit-tool-details>
+
+    <div class="row">
+        <div class="col" style="float:none">
+            <input-text
+                v-model="geomarkServiceUrl"
+            >Geomark service URL</input-text>
+        </div>
+    </div>
+    <div class="row">
+        <input-checkbox class="col s6"
+            v-model="enableCreateFromFile"
+        >Enable link to create a geomark from a file</input-checkbox>
```

Comment:
Since we're moving away from links, maybe something like 'Show create geomark from file option' or something like that?

Reply:
I'll update this to avoid mention of links.